kubectl-gadget.Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG BUILDER_IMAGE=golang:1.18-bullseye
```

Comment:
What is the use case of this Dockerfile?
To enable having a portable `kubectl-gadget` where user only needs to give the `.kubeconfig` to interact with an existing cluster?

Reply:
Yes, it's another way to ship kubectl-gadget, if you already have docker or a similar runtime you can just pull the container image and run kubectl-gadget from there without having to download the binary by hand.